packages/node/src/evm/handlers/initialize-provider.test.ts

```diff
@@ -15,6 +15,7 @@ import * as fixtures from '../../../test/fixtures';
 
 describe('initializeProvider', () => {
   it('fetches, maps and authorizes requests', async () => {
+    jest.setTimeout(10000);
```

Comment:
Do we need this here? Things worked previously without it?

Reply:
I will revert this. When running the whole suite on my local env, this test consistently failed with a 5sec timeout but that's probably due to not having enough resources on my computer.